shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -56,36 +56,52 @@ namespace electron {
 
 namespace api {
 
-typedef std::unordered_map<content::RenderFrameHost*, WebFrameMain*>
-    RenderFrameMap;
-base::LazyInstance<RenderFrameMap>::DestructorAtExit g_render_frame_map =
-    LAZY_INSTANCE_INITIALIZER;
-
-WebFrameMain* FromRenderFrameHost(content::RenderFrameHost* rfh) {
-  auto frame_map = g_render_frame_map.Get();
-  auto iter = frame_map.find(rfh);
+typedef std::unordered_map<int, WebFrameMain*> WebFrameMainIdMap;
+
+base::LazyInstance<WebFrameMainIdMap>::DestructorAtExit
+    g_web_frame_main_id_map = LAZY_INSTANCE_INITIALIZER;
```

Comment:
Per https://source.chromium.org/chromium/chromium/src/+/main:base/lazy_instance.h;l=5-13?q=base::LazyInstance&ss=chromium.

```
// Please don't introduce new instances of LazyInstance<T>. Use a function-local
// static of type base::NoDestructor<T> instead:
//
// Factory& Factory::GetInstance() {
//   static base::NoDestructor<Factory> instance;
//   return *instance;
// }
```

Reply:
Thanks for noticing this! I'll have this updated soon.